emission/analysis/intake/cleaning/location_smoothing.py

```diff
@@ -109,11 +109,19 @@ def add_heading_change(points_df):
 def filter_current_sections(user_id):
     time_query = epq.get_time_range_for_smoothing(user_id)
     try:
+        ts = esta.TimeSeries.get_time_series(user_id)
         sections_to_process = esda.get_entries(esda.RAW_SECTION_KEY, user_id,
                                                time_query)
+        time_query.timeType = "data.ts"
```

Comment:
note to self: this is because before this change, we used to read the locations per section using `data.ts`

https://github.com/e-mission/e-mission-server/blob/823d47f062125a82e48276fcd542095bbb773d61/emission/storage/decorations/analysis_timeseries_queries.py#L103